.circleci/config.yml

```diff
@@ -45,12 +45,7 @@ jobs:
       - checkout
       # database in neighbor container need some time to start
       # wait for the DB to be reachable, up to 15 pings with 1 second interval
```

Comment:
I believe this comment should be moved to the scripts. Here the reader does not need know about such intricacies.